app/assets/stylesheets/module-dashboards.scss

```diff
@@ -166,6 +166,18 @@
           padding-right: 20px;
         }
       }
+
+      &-text {
+        display: block;
+      }
+
+      &.padding-right .dashboards-home-main--th-text {
+        padding-right: 1em;
```

Comment:
If I remember there are a set of CSS helpers (quite oldies) to do so. In this case, it could be `pr-1` or similar